common/setups/rasr/gmm_system.py

```diff
@@ -1087,19 +1087,19 @@ def get_gmm_output(
                 corpus_key
             ].get(scorer_key, [None])[-1]
 
-        if corpus_type == "dev" and self.alignments[corpus_key].get(
-            f"alignment_{steps.get_step_names_as_list()[step_idx - 1]}", False
-        ):
+        if (corpus_type == "dev" or corpus_type == "test") and self.alignments[
```

Comment:
usually we don't have alignments for the test sets, because we "assume" that the transcription is unknown..

Reply:
I agree that we usually dont have that, but I still think it should be supported for cases like TTS or when you would want to do some statistics etc. where you need an alignment.

Reply:
ok, we can keep this